nixos/modules/services/network-filesystems/ipfs-cluster.nix

```diff
@@ -82,6 +82,7 @@ in {
       wantedBy = [ "default.target" ];
 
       serviceConfig = {
+        # "" clears exec list (man systemd.service -> execStart)
```

Comment:
This is such an obscure thing to do, serioiusly `systemd`... I had to read carefully to find that, and it seems it does not even apply to all list-type options.
https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=